### PR TITLE
Create dhd.csl

### PR DESCRIPTION
**Pull request by fussballlinguist**

### CODE REVIEW

**Comment by fussballlinguist** [comment]
Hi, I have one issue with the style. I would like to include the collection title and number into the title, so that the full stop appears only after the brackets.

Now it looks like this:

Mustermann, Max (2021): "Kapiteltitel" in: Gates, Bill / Jobs, Steve (eds.): Buchtitel. (= Reihentitel 5) Musterstadt: Musterverlag 1–20. 10.15/00001.

But I wish to look it like this:

Mustermann, Max (2021): "Kapiteltitel" in: Gates, Bill / Jobs, Steve (eds.): Buchtitel (= Reihentitel 5). Musterstadt: Musterverlag 1–20. 10.15/00001.

File: dhd.csl [hunk 1]
```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="de-DE">
+  <info>
+    <title>DHd (Deutsch)</title>
+    <title-short>DHd (Deutsch)</title-short>
+    <id>http://www.zotero.org/styles/dhd</id>
```
Comment (bwiernik) [inline dhd.csl:6]: Please spell out DHd in full in the title and ID. Please rename the CSL file accordingly

File: dhd.csl [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="creator-year-long">
+    <choose>
+      <if match="none" variable="author editor">
+        <text value="o. A." font-weight="bold" suffix=" "/>
+      </if>
+    </choose>
```
Comment (bwiernik) [inline dhd.csl:35]:
```suggestion
```

File: dhd.csl [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    <group delimiter=" ">
+      <names variable="author" font-weight="normal">
+        <name font-weight="bold" delimiter=" / " name-as-sort-order="all"/>
+        <label form="short" prefix=" (" suffix=")"/>
+        <substitute>
+          <names variable="editor"/>
```
Comment (bwiernik) [inline dhd.csl:41]:
```suggestion
          <names variable="editor"/>
          <text term="anonymous" form="short"/>
```

File: dhd.csl [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+              <text variable="title" prefix="&quot;" suffix="&quot;"/>
+              <text macro="editor-container-title"/>
+              <text macro="collection-title-number"/>
+              <text macro="edition"/>
+              <text macro="publication-place-publisher"/>
+              <text macro="page-range"/>
+              <text variable="DOI"/>
+            </group>
+          </else-if>
+          <else-if type="dataset" match="any"/>
```
Comment (bwiernik) [inline dhd.csl:167]: Is it intended that datasets are omitted from the bibliography?

**Review by bwiernik — CHANGES_REQUESTED** [review]

**Comment by bwiernik** [comment]
To move the series title and number before the full stop, put those variables inside the container title macro.

File: dhd.csl (hunk 1 above)
Comment (fussballlinguist) [reply dhd.csl]: Done…

File: dhd.csl (hunk 4 above)
Comment (fussballlinguist) [reply dhd.csl]: I would like to leave this open to the Germanophone DH community who will meet in two weeks to discuss the requirements

**Comment by bwiernik** [comment]
Okay, this looks good to me overall. Please let me know when you have finished your discussions and are ready to merge.

**Comment by fussballlinguist** [comment]
> Okay, this looks good to me overall. Please let me know when you have finished your discussions and are ready to merge.

We skip the dataset issue for a moment, I deleted the code snippet from the csl-file which is now ready to merge. Thanks!

**Comment by bwiernik** [comment]
Thanks!

**Pull request merged**